--- tsk/img/ewf.c.orig	2016-10-31 11:04:55.857626481 +0900
+++ tsk/img/ewf.c	2016-10-31 11:06:25.776628726 +0900
@@ -64,7 +64,7 @@
 
     tsk_take_lock(&(ewf_info->read_lock));
 #if defined( HAVE_LIBEWF_V2_API )
-    cnt = libewf_handle_read_random(ewf_info->handle,
+    cnt = libewf_handle_read_buffer_at_offset(ewf_info->handle,
         buf, len, offset, &ewf_error);
     if (cnt < 0) {
         char *errmsg = NULL;
